Jjaschke

jjaschke98 opened the pull request:
getting latlng info from Strava API using the routeID

** can somebody look over the error handling and line 147 in content.js where I am trying to return the array of latlng but it is a promise so I am struggling with that. 

** also @terrnance I changed the routeID config a bit to set the route that the user selects so I changed the way the id is set and got a new route_id in the onClick function - I am not sure if this is going to be an issue with anything you had planned previously 


** additionally we need a way to deal with client secret and client id (removed from what I am committing)

Commit:
setting up strava api

Commit:
Merge branch 'master' into jjaschke

Commit:
adding strava api

Commit:
merge conflicts

Commit:
Merge branch 'master' into jjaschke

Commit:
working on getting latlng from promise

Commit:
merge conflicts

Commit:
incorporated selected routeID

terrnance committed:
Checking over Promises

Promises looked good, I only deleted only extra .then which was ".then(res => res)" after you already called the getLatandLog function. I also just added curly braces and return statements in order to make it more clear what the promises were doing.

terrnance reviewed (commented):
Promises looked good, I only deleted only extra .then which was ".then(res => res)" after you had already called the getLatandLog function. I also just added curly braces and return statements in order to make it more clear what the promises were doing.

terrnance committed:
Await/Async Notation

I also forgot the await and async notation for functions that include promises. Without this the actual array of longitudes and latitudes won't be displayed/properly returned to displayPreview. It will only show a promise.

terrnance commented:
I also forgot the await and async notation for functions that include promises. Without this the actual array of longitudes and latitudes won't be displayed/properly returned to displayPreview. It will only show a promise.

terrnance committed:
Don't need to commit the client id and client secret
